YvesIraguha/TeaHouse
# #169776878 CRUD Book Collections

#### What does this PR do?
- Implement CRUD operations on Book Collections
#### Description of Task to be completed?
- write tests for book collections
- create functionalities
- enable saving files and images on cloudinary
#### How should this be manually tested?
- Clone the repo and checkout the branch `upload-read-collection-169776878`
- Update your `.env` file with fields specified in `.env.sample` file. 
- Run `npm i && npm run precommit && npm run start` linting, and the test should pass successfully, and the server should immediately start. 
- Consume the endpoints related to collections, with correct data and authorization. Then, you should see the outcome as expected. 
#### Any background context you want to provide?
- So far a user can update the title, type, and author/creator of the book collection.
#### What are the relevant pivotal tracker stories?
#[169776878](https://www.pivotaltracker.com/story/show/169776878)
